geotrellis/geotrellis-server
# Include opengis in `root` project

## Overview

Adds `opengis` to the `root` project to prevent some head-scratching when publishing local for development.

### Checklist


## Testing Instructions

- Start an sbt shell, run `publishLocal` and see that artifacts for `opengis` are also published
